packages/core-utils/package.json

```diff
@@ -20,6 +20,7 @@
     "chroma-js": "^2.4.2",
     "date-fns": "^2.28.0",
     "date-fns-tz": "^1.2.2",
+    "graphql-tag": "^2.12.6",
```

Comment:
We might actually not need graphql-tag at all. I think this was only needed if we were using `gql` tag in the code, which we are not, since the GQL query is in an external file.

Reply:
Agreed. We can remove it then!